client/src/index.js

```diff
@@ -3,11 +3,19 @@ import ReactDOM from "react-dom";
 import App from "./App";
 import * as serviceWorker from "./serviceWorker";
 import { UserContextProvider } from "./contexts/user/UserContextProvider";
+import { Elements } from "@stripe/react-stripe-js";
+import { loadStripe } from "@stripe/stripe-js";
+
+const stripePromise = loadStripe(
+    "pk_test_51HrOLODpMZzy7YFf2ImPzBZIAj7wyO69oVlA2hyu7e7BuLctRAzEVxTAnLwkRmKHdUgLW4SRjPpnPvxJyJMAumHq00PNeexRAM"
```

Comment:
Should this be in a .env?

Reply:
Yes, it probably should. I will add that in a future commit (Should be safe for now since it's only a test key)